lib/plist-transform.js

```diff
@@ -31,9 +32,29 @@ module.exports = (dir, pkg) => {
 		.then(content => {
 			const data = fix(plist.parse(content));
 			data.version = pkg.version || '';
-			data.description = pkg.description || '';
-			data.webaddress = pkg.homepage || pkg.author.url || '';
-			data.createdby = pkg.author.name || '';
+			data.description = data.description || pkg.description || '';
```

Comment:
Why change the current behaviour? This is done on purpose, we always want the description from `package.json` to keep them in sync. It wouldn't make sense to set 2 different descriptions.